packages/angular_devkit/build_angular/src/angular-cli-files/plugins/common-js-usage-warn-plugin.ts

```diff
@@ -82,7 +82,8 @@ export class CommonJsUsageWarnPlugin {
             // will require CommonJS libraries for live reloading such as 'sockjs-node'.
             if (mainIssuer?.name === 'main' && !issuer?.userRequest?.includes('webpack-dev-server')) {
               const warning = `${issuer?.userRequest} depends on ${rawRequest}. CommonJS or AMD dependencies can cause optimization bailouts.\n` +
-                'For more info see: https://web.dev/commonjs-larger-bundles';
+                'For more info see: https://web.dev/commonjs-larger-bundles\n' +
+                `To disable this warning, add "${rawRequest}" to the "allowedCommonJsDependencies" option in angular.json`;
```

Comment:
Can we add a full-stop at the end of the sentence.

Reply:
@alan-agius4 Sure, done 👍